### ISSUE TITLE
Cyberpunk crash with NVIDIA GTX  1080Ti

### ISSUE THREAD

**Issue by Vincenzo-Puzone** [HansKristian-Work/vkd3d-proton#1930]
Please describe your issue as accurately as possible. Include screenshots or videos if relevant
Trying various version of Proton (Even GE) and Wine with lutris but the game simply doesn't want to start (other games affected are Tom Clancy's The Division 2)

### Software information
Name of the game, settings used etc.
Cyberpunk 2077, tried every env vars found across other issues
### System information
- GPU:Nvidia GTX 1080Ti
- Driver:550.40.53
- Wine version: Last one Wine-GE 8.26, tried Pronton Experimental,9.0,8.0-5
- VKD3D-Proton version: 2.6 through 2.11.1

### Log file
[game-log.txt](https://github.com/HansKristian-Work/vkd3d-proton/files/14515585/game-log.txt)
dmesg
```console
NVRM: Xid (PCI:0000:09:00): 69, pid='<unknown>', name=<unknown>, Class Error: ChId 00a7, Class 0000c197, Offset 00002380, Data 00000000, ErrorCode 00000004
```

**Comment by yslai** [comment]
This problem seems to be specific to the Pascal generation (e.g. does not occur with Turing cards) and is a regression between the Nvidia proprietary driver versions 530 and 535. For example, driver 530.41.03 does not trigger the Xid 69.

But since this last functioning driver version is affected by CVE‑2024‑0074 and CVE‑2024‑0075, there are no safe versions to run Cyberpunk 2077 on Pascal anymore.

**Comment by ArklandHan** [comment]
I also have a trusty 1080ti and have been attacking this same problem all week. Nothing I've tried so far allows Cyberpunk to run on any current proton (that I have tried), but it seems to run on Proton 7.0-6 or up to GE-Proton7-42 (w/ steam). No success with anything after that. Runs a little slower than when I boot windows, but it's playable so far. 

Launch options:
VKD3D_CONFIG=force_static_cbv DXVK_FRAME_RATE=60 gamemoderun %command% --launcher-skip

**Comment by yslai** [comment]
After testing (and remembering some older configuration that works) I found out that without `dxgi=n`, the current [Wine-GE-Proton8-26](https://github.com/GloriousEggroll/wine-ge-custom/releases/tag/GE-Proton8-26) can run with older VKD3D-proton. Also note that I am using a 1080Ti in eGPU configuration and with Prime render offload. Additionally, I wrote a script that automates somewhat the building (particularly patching all the old `git://` protocols in the older `.gitmodules`) to make a proper bisect possible.

I found that unlike this issue and #1869, tag `v2.5`, `v2.6`, `v2.7` all work for me. The first offending tag is `v2.8`. Bisecting then narrows the problematic commit down to d0580cb6e3c63424e92c4f787c8d1d00b953826b.

**Comment by ArklandHan** [comment]
`VKD3D_DISABLE_EXTENSIONS` worked like a charm.

**Comment by esullivan-nvidia** [comment]
Would one of you be able to try the latest Vulkan developer beta driver (550.40.65) and confirm if the issue observed with VK_EXT_descriptor_buffer is resolved? You can download the run file for the driver here:

https://developer.nvidia.com/vulkan-driver

**Comment by ArklandHan** [comment]
Sorry. I would, but the RAM in the computer with that GPU went bad, and I used that as an excuse to put a new build together, so I'm not using the 1080ti anymore.

**Comment by aa889788** [comment]
I've met the same problem with any dx12 games, the gpu is Tesla P4, I've tried 535.146, 535.183, 550.54, 555.58, none of the makes the game work

**Comment by Blisto91** [comment]
@yslai @aa889788 @Vincenzo-Puzone Was this issue solved for you with newer drivers?

**Comment by yslai** [comment]
Yes, I subsequently posted the information I gathered in the Linux Driver part of the Nvidia developer forum (https://forums.developer.nvidia.com/t/vk-ext-descriptor-buffer-in-vkd3d-triggers-xid-69-on-pascal-cards-since-driver-535-series/291172), and there was a response by an Nvidia employee referencing an internal ticket. A few driver versions thereafter the bug was fixed. I have not run an up-to-date test recently, and keep in mind that Xid error codes are rather generic and difficult to root-cause without a deeper analysis.
